packages/instantsearch.js/src/widgets/search-box/__tests__/search-box.test.tsx

```diff
@@ -65,6 +65,7 @@ describe('searchBox', () => {
           height="10"
           viewBox="0 0 40 40"
           width="10"
+          aria-hidden="true"
```

Comment:
is there a forgotten commit? I can see all the test updates, but the code didn't change 🤔

Reply:
react-instantsearch-hooks-web/ui is updated, but not React InstantSearch dom / components, InstantSearch.js / components, Vue InstantSearch / components :)

Reply:
You're right 😅 I did a search for exact match classnames, but it looks like some are dynamically built. Thanks for catching. Will update soon.